sdk/keyvault/azcertificates/go.mod

```
@@ -1,3 +1,4 @@
+// Deprecated: The latest supported version of this library is at github.com/Azure/azure-sdk-for-go/sdk/security/keyvault/azcertificates
```

The deprecation messages can simply tell people what to do:
```suggestion
// Deprecated: use github.com/Azure/azure-sdk-for-go/sdk/security/keyvault/azcertificates instead
```